includes/Core/Authentication/Previous_Connected_Proxy_URL.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Authentication\Previous_Connected_Proxy_URL
+ *
+ * @package   Google\Site_Kit\Core\Authentication
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Authentication;
+
+/**
+ * Previous_Connected_Proxy_URL class.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Previous_Connected_Proxy_URL extends Connected_Proxy_URL {
+
+	/**
+	 * The option_name for this setting.
+	 */
+	const OPTION = Connected_Proxy_URL::OPTION . '_previous';
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+		add_filter(
```

Comment:
The `update_option_{name}` is an action, not a filter:
```suggestion
		add_action(
```

Reply:
https://github.com/google/site-kit-wp/pull/4536/commits/1650e8044c8706172b1d75c9952c467b0650eca6